src/runtime/pkg/katautils/config.go

```diff
@@ -941,7 +907,6 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		RootfsType:               rootfsType,
 		FirmwarePath:             firmware,
 		FirmwareVolumePath:       firmwareVolume,
-		SnpCertsPath:             snpCertsPath,
```

Comment:
Can this field be removed from Hypervisor config now too: https://github.com/kata-containers/kata-containers/blob/d7059e9024d456bb81aaa974eff0bd9e4cc1d977/src/runtime/virtcontainers/hypervisor.go#L463-L466 ?

Reply:
Oops. Missed that one. Re-testing now.